doc/user_manual/sampler.tex

```diff
@@ -401,6 +401,91 @@ \section{Samplers}
         \texttt{out\_100.pk}.
     \end{itemize}
 }
+
+\newcommand{\convergenceDescriptionAMC}
+{
+  \xmlNode{Convergence} recognizes the following child nodes:
+    \begin{itemize}
+      \item \xmlNode{limit}, \xmlDesc{integer required field}, the
+        maximum number of adaptive samples (iterations).
+        %
+        \default{infinite}.
+      \item \xmlNode{forceIteration}, \xmlDesc{boolean optional field},
+        this attribute controls if at least a number of iterations equal to
+        \textbf{limit} must be performed.
+        %
+        \default{False}.
+      \item \xmlNode{persistence}, \xmlDesc{integer optional field},
+        offers an additional convergence check.
+        %
+        It represents the number of times the computed error needs to be
+        below the inputted tolerance before convergence is reported.
+        %
+        \default{5}.
+
+      \item \xmlNode{"metric"}, \xmlDesc{comma separated string list, required field},
+        specifications for the metric to be calculated.  The name of each node is the requested metric. The text of the node is a comma-separated list of the
+        parameters for which the metric should be calculated. See the example below.\\
+
+
+        RAVEN will automatically calculate the standard errors on some scalar quantities. Currently the quantities available for request are:
+        \begin{itemize}
+          \item \textbf{expectedValue}: expected value or mean
+          \item \textbf{median}: median
+          \item \textbf{variance}: variance
+          \item \textbf{sigma}: standard deviation
+          \item \textbf{skewness}: skewness
+          \item \textbf{kurtosis}: excess kurtosis (also known as Fisher's kurtosis)
+        \end{itemize}
+
+
+
+      This XML node needs to contain the attribute:
+      \begin{itemize}
+        \itemsep0em
+        \item \xmlAttr{prefix}, \xmlDesc{required string attribute}, user-defined prefix for the given \textbf{metric}.
+          For scalar quantifies, RAVEN will define a variable with name defined as:  ``prefix'' + ``\_'' + ``parameter name''.
+          For example, if we define ``mean'' as the prefix for \textbf{expectedValue}, and parameter ``x'', then variable
+          ``mean\_x'' will be defined by RAVEN.
+          For matrix quantities, RAVEN will define a variable with name defined as: ``prefix'' + ``\_'' + ``target parameter name'' + ``\_'' + ``feature parameter name''.
+          For example, if we define ``sen'' as the prefix for \textbf{sensitivity}, target ``y'' and feature ``x'', then
+          variable ``sen\_y\_x'' will be defined by RAVEN.
+          \nb These variable will be used by RAVEN for the internal calculations. It is also accessible by the user through
+          \textbf{DataObjects} and \textbf{OutStreams}.
+
+          \item \xmlAttr{tol}, \xmlDesc{required float attribute}, convergence tolerance for the metric.
```

Comment:
More description ... Something like "The convergence is reported when the relative error between the current and previous iterations falls below this value"?